handlers/runCmd.ts

```diff
@@ -81,16 +83,37 @@ export const RunCmd = async (
 
   denoCmd.push(appFile);
 
-  let appStatus = await Deno.run({
-    cmd: denoCmd,
-  }).status();
+  // * reload project when files changes
+  const runApp = (): Deno.Process => {
```

Comment:
@buttercubz Is this fully working? The app can get rebooted multiple times and it will still work? I'm asking this because I'm afraid that if a file changes, the process is closed but not killed thus leaving the port open so when the app tries to reboot it will throw a network error since the port is in use. Not sure, let me know if this is fully working